enginetest/queries.go

```diff
@@ -3245,13 +3245,13 @@ var QueryTests = []QueryTest{
 		},
 	},
 	{
-		Query: `SHOW VARIABLES LIKE 'gtid_mode`,
+		Query: `SHOW VARIABLES LIKE 'gtid_mode'`,
```

Comment:
These queries were busted and only worked because the adhoc lexer didn't error on the unterminated string literal.